quantus/metrics/axiomatic/completeness.py

```diff
@@ -65,7 +73,7 @@ def __init__(
         normalise_func_kwargs: Optional[Dict[str, Any]] = None,
         output_func: Optional[Callable] = lambda x: x,
```

Comment:
using mutable default values for parameters of mutable type is dangerous:
https://florimond.dev/en/posts/2018/08/python-mutable-defaults-are-the-source-of-all-evil/